### PR TITLE
update to ganache v7.7.7

### PR DESCRIPTION
**Pull request by davidmurdoch**
This PR also adds a link to ganache.dev in truffle's `--help` output. On hold, wait until v7.7.7 is released before merging.

### CODE REVIEW

File: packages/core/lib/command-utils.js [hunk 1]
```diff
@@ -324,7 +324,9 @@ const displayGeneralHelp = options => {
         OS.EOL +
         "Usage: truffle <command> [options]"
     )
-    .epilog("See more at http://trufflesuite.com/docs")
+    .epilog("See more at https://trufflesuite.com/docs/" + 
+      OS.EOL +
+      "For Ethereum JSON-RPC documentation see https://ganache.dev")
```
Comment (gnidan) [inline packages/core/lib/command-utils.js:329]: 👌

**Review by jeffsmale90 — APPROVED** [review]
Looks reasonable to me, but what do I know :shrug:

**Review by cds-amal — APPROVED** [review]
Thanks, @davidmurdoch . Looks good!

File: packages/core/lib/command-utils.js (hunk 1 above)
Comment (cds-amal) [reply packages/core/lib/command-utils.js:329]: Ok!

**Pull request merged**